tests/data/console/controllers/fixtures/subdir/SecondFixture.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace yiiunit\data\console\controllers\fixtures\subdir;
+
+use yii\test\Fixture;
+use yiiunit\data\console\controllers\fixtures\FixtureStorage;
+
+class SecondFixture extends Fixture
+{
+    public function load()
+    {
+        FixtureStorage::$subdirSecondFixtureData[] = 'some data set for subdir/second fixture';
```

Comment:
> more understandable description 'dummy string'

Agree. But still, I can't see any way this should've been guessed from all the other fixture contents.
I mean, check out any of the `tests/data/console/controllers/fixtures/*Fixture.php` - they all have their data in format of `"some data set for <fixture-name-here> fixture"`...
Hence the question: fixture content should be replaced to `dummy string` only in *added* fixtures, or in *all* of the mentioned fixtures?